### PR TITLE
Added support for Bit in peak call args

### PR DESCRIPTION
**Pull request by jack-melchert**

### CODE REVIEW

File: peak/mapper/utils.py [hunk 1]
```diff
@@ -252,17 +252,29 @@ def solved_to_bv(var, solver):
     else:
         return BitVector[var.size](solver_value)
 
+def rebind_value(val, _family):
```
Comment (cdonovick) [inline peak/mapper/utils.py:255]: This type of code makes me think I should define the families likes:
```Python
class _VarientFamily:
    ...
VarientFamily = _VarientFamily()
```

So there doesn't need to be so much construction
Although that would make building extend families more tricky
As we can currently do something like:
```Python
class AbsFloatFamily(AbstractFamily):
    @property
    @abstractmethod
    def Float(self): pass

class PyFloatFamily(PyFamily, AbstractFamily):
    @property
    def Float(self):
        return hwtypes.FPVector
```

But this has nothing to do with is PR.

Comment (cdonovick) [reply peak/mapper/utils.py:255]: Oh also its more pythonic to use `var_` than `_var` when`var` is already taken but I don't really care.

Comment (cdonovick) [reply peak/mapper/utils.py:255]: as `_var` usually implies `_var` is an implementation detail.

Comment (rdaly525) [reply peak/mapper/utils.py:255]: Noted on the variable naming

**Pull request merged**